CryptoPkg/Library/BaseCryptLib/Pk/CryptX509.c

```diff
@@ -434,6 +434,17 @@ InternalX509GetNIDName (
   }
 
   EntryData = X509_NAME_ENTRY_get_data (Entry);
+  // MU_CHANGE [BEGIN] - CodeQL change
+  if (EntryData == NULL) {
+    //
+    // Fail to retrieve name entry data
+    //
+    *CommonNameSize = 0;
+    ReturnStatus    = RETURN_NOT_FOUND;
+    goto _Exit;
+  }
+
+  // MU_CHANGE [END] - CodeQL change
 
   Length = ASN1_STRING_to_UTF8 (&UTF8Name, EntryData);

```

Comment:
Technically, this was handled before because `ASN1_STRING_to_UTF8` returns `-1` if the second argument is null. But, I think this is fine to satisfy CodeQL.

Reply:
This is an inaccurate failure I don't want to add this to the SM02311 exception list in case there are additional legitimate issues.